IntegrationTests/UI.IntegrationTests/UITest.cs

```diff
@@ -33,42 +33,30 @@ void HandleApplicationIdle(object sender, EventArgs e)
             Func<T, Task> runTestAsync)
             where T : Form
         {
-            // If form.Dipose was called by the async test task, closing the window would be done in a strange order.
+            Assert.IsEmpty(Application.OpenForms.OfType<T>(), $"{Application.OpenForms.OfType<T>().Count()} open form(s) before test");
```

Comment:
📝 I originally had these assertions checking `Application.OpenForms`, but that was a stronger assertion than necessary for these tests.

Reply:
reasonable because more generic